backend/app/api/src/endpoints/organization/dashboard/organization/PatchOrganizationEndpoint.test.ts

```diff
@@ -59,4 +66,1245 @@ describe('Endpoint.PatchOrganization', () => {
+
+                // act
+                const response = await patchOrganization({
+                    patch,
+                    organization,
+                    token,
+                });
+
+                // assert
+                expect(response.body).toBeDefined();
+                expect(response.body.privateMeta).not.toBeNull();
+
+                // roles
+                expect(response.body.privateMeta!.roles).toHaveLength(1);
+                expect(response.body.privateMeta!.roles[0].id).toBe(role1.id);
+                expect(response.body.privateMeta!.roles[0].resources.size).toBe(1);
+
+                // responsibilities
+                expect(response.body.privateMeta!.responsibilities).toHaveLength(1);
+                expect(response.body.privateMeta!.responsibilities[0].id).toBe(memberResponsibility.id);
+                expect(response.body.privateMeta!.responsibilities[0].permissions).not.toBeNull();
+                expect(response.body.privateMeta!.responsibilities[0].permissions?.resources.size).toBe(1);
+
+                // inherited responsibility roles
+                expect(response.body.privateMeta!.inheritedResponsibilityRoles).toHaveLength(1);
+                expect(response.body.privateMeta!.inheritedResponsibilityRoles[0].id).toBe(inheritedResponsibilityRole1.id);
+                expect(response.body.privateMeta!.inheritedResponsibilityRoles[0].resources.size).toBe(1);
+            });
+        });
+
+        describe('full access to webshop', () => {
+            let user: User;
+            let token: Token;
+            let organization: Organization;
+            let webshop: Webshop;
+
+            beforeAll(async () => {
+                const createWebshopRole = PermissionRoleDetailed.create({
+                    accessRights: [AccessRight.OrganizationCreateWebshops],
+                });
+
+                organization = await new OrganizationFactory({ roles: [createWebshopRole] }).create();
+                webshop = await new WebshopFactory({ organizationId: organization.id }).create();
+
+                // organization.privateMeta.roles = [createWebshopRole];
+                // await organization.save();
+
+                user = await new UserFactory({
+                    organization,
+                    permissions: Permissions.create({ level: PermissionLevel.None, roles: [createWebshopRole], resources: new Map([
+                        [PermissionsResourceType.Webshops,
+                            new Map([
+                                [webshop.id,
+                                    ResourcePermissions.create({ level: PermissionLevel.Full }),
+                                ],
+                            ])],
+                    ]) }),
+                })
+                    .create();
+
+                token = await Token.createToken(user);
+            });
+
+            test('should be able to put limited roles', async () => {
```

Comment:
Putting roles is not allowed (see other comments), so test should test for error instead